features/kinderen.feature

```diff
@@ -64,10 +64,10 @@ Functionaliteit: Kinderen van een persoon raadplegen
   Rule: Een kind wordt alleen teruggegeven als minimaal één gegeven in de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) van het kind een waarde heeft.
     - Wanneer in een categorie kind alleen gegevens zijn opgenomen in groep 81 of 82, 85 en 86, wordt dit kind niet opgenomen in het antwoord
     - Wanneer een gegeven een standaardwaarde heeft, zoals "." (punt) bij geslachtsnaam of "00000000" bij geboortedatum, geldt dat hier als het bestaan van een waarde en wordt het kind wel geleverd
-    - Wanneer door de gebruikte fields parameter in het request het kind in de response geen enkel gegeven heeft met een waarde, dan wordt het kind niet geleverd
+    - Wanneer door de gebruikte fields parameter in het request het kind in de response geen enkel gegeven heeft met een waarde, dan wordt het kind geleverd zonder gegevens (dus als leeg object)
 
     @gba
-    Abstract Scenario: kind volledig onbekend
+    Abstract Scenario: kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields
       Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende kinderen in de registratie
```

Comment:
Moet hier niet het bsn 999996150 vermeld worden ipv 555550001? Dat staat iig in regel 77 aangegeven.

Reply:
Klopt. Aangepast

Reply:
@JohanBoer gaat er niet iets mis? Je zegt dat het is aangepast maar ik zie de aanpassingen niet terug. Ook de aanpassingen n.a.v. opmerkingen van @fsamwel zie ik niet.

Reply:
Nu wel!

Reply:
Correctie, in regel 115 staat volgens mij nog steeds de verkeerde bsn.

Reply:
Aangepast